frontends/web/src/routes/device/bitbox02/checkbackup.tsx

```diff
@@ -44,46 +45,39 @@ class Check extends Component<Props, State> {
     userVerified: false,
   };
 
-  private checkBackup = async (silent: boolean, backups: Backup[]) => {
-    const response = await bitbox02API.checkBackup(this.props.deviceID, silent);
-    let message;
-    // "silent" call gets the backup id from device without blocking to display in dialogue
-    if (silent && response.success && response.backupID) {
-      const foundBackup = backups.find((backup: Backup) => backup.id === response.backupID);
-      if (foundBackup) {
-        this.setState({ foundBackup });
-        message = this.props.t('backup.check.success');
-      } else {
-        message = this.props.t('unknownError', { errorMessage: 'Not found' });
+  private checkBackup = async () => {
+    try {
+      const backupID = await bitbox02API.checkBackup(this.props.deviceID, true);
+      const foundBackup = this.props.backups.find((backup: Backup) => backup.id === backupID);
+      if (!foundBackup) {
+        alertUser(this.props.t('unknownError', { errorMessage: 'Not found' }));
```

Comment:
The alertUser global component is a bit a react antipattern, why not reuse the dialog that is here?

for example:

```js
this.setState({
  activeDialog: true,
  userVerified: true,
  message: this.props.t('unknownError', { errorMessage: 'Not found' })
});
```

but yes, `userVerified` doesn't make too much sense. But this could be renamed from `userVerified` to `userVerify`

Reply:
@thisconnect I chose it because the alert looks just like a dialog, but the message is in the body instead of in the title. I had your proposed code before and the message was in the dialog title with an empty body, which looks strange.

Why is it an antipattern?

Reply:
ack, it's fine we use it in various places..

This is what we should change one day: 

At the moment alertUser is a variable in a global scope pointing to a function of the <Alert /> instance. This "works" 
but is a bit a hack and not react style.

https://github.com/digitalbitbox/bitbox-wallet-app/blob/453eb3935c114408d28051b726714a7e4b1806f1/frontends/web/src/components/alert/Alert.tsx#L49